core/src/OpenACC/Kokkos_OpenACC_Traits.hpp

```diff
@@ -22,8 +22,7 @@
 namespace Kokkos::Experimental::Impl {
 
 struct OpenACC_Traits {
-#if defined(KOKKOS_ARCH_PASCAL) || defined(KOKKOS_ARCH_VOLTA) || \
-    defined(KOKKOS_ARCH_AMPERE) || defined(KOKKOS_ARCH_HOPPER)
+#if defined(KOKKOS_IMPL_ARCH_NVIDIA_GPU)
```

Comment:
This is a different condition than before. You are adding Kepler and Maxwell to the `if`, is it ok to do it?

Reply:
You are correct but the intent was to enable for NVIDIA GPUs so the new code is doing the right thing